### DISCUSSION TITLE
Gravatar support

### DISCUSSION THREAD

**Discussion by mpfl** [lldap/lldap#622] [Ideas]
Having finally implemented LLDAP, I see that there is support for user profile images. Rather than uploading them manually, what if LLDAP could slurp them from Gravatar?

**Comment by nitnelave** [comment]
Hi! Since this is more of a one time thing, it would complexify LLDAP more than it's worth, I think. However, with the graphql api, it shouldn't be too hard to write a script that queries LLDAP for the user info, gets the avatars from gravatar, and uploads them back into LLDAP.
